src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public List<String> readFromFile(String fileName) {
+        List<String> dataFromFile = new ArrayList<>();
+        File file = new File(fileName);
+        if (file.exists()) {
```

Comment:
if file does not exist, we will return just an empty list - did you mean this?

Reply:
in this method we should return List, if we didn’t find file, we’ll return an empty list. Is it a bad practice?

Reply:
if it is your intention, then ok